src/seedu/addressbook/data/person/Gender.java

```diff
@@ -0,0 +1,60 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+
+public class Gender {
```

Comment:
Coding standard requires to include descriptive header comments for public classes